features/filter_dimension.authorised.feature

```diff
@@ -117,6 +117,7 @@ Feature: Filter Outputs Private Endpoints Enabled
           "name": "geography",
           "id": "city",
           "label": "City",
+          "default_categorisaion":  "",
```

Comment:
Adding these blank values is no different from omitting completely. You should add a value if you want to capture any new functionality.

Reply:
Nick mentioned that he needed a clear signal for the frontend controllers to understand when there is a default categorisation, so we thought of including it always, with the empty string representing a nil value, thus no default categorisation.